amethyst_renderer/src/lib.rs

```diff
@@ -265,7 +265,8 @@ fn init_backend(wb: WindowBuilder, el: &EventsLoop) -> Result<Backend> {
 
     let wb = glutin::WindowBuilder::from_winit_builder(wb)
         .with_gl_profile(GlProfile::Core)
-        .with_gl(GlRequest::Latest);
+        .with_gl(GlRequest::Latest)
```

Comment:
Could we have a temporary workaround for https://github.com/tomaka/glutin/issues/861 here?

Maybe something like this:

```rust
    .with_pixel_format(24, 8)
    .with_gl(glutin::GlRequest::GlThenGles {
        opengles_version: (2, 0),
        opengl_version: (2, 1),
})
```